tapdb/sqlc/migrations/000015_tapscript_trees.down.sql

```diff
@@ -0,0 +1,2 @@
+-- This file is empty on purpose. There is nothing to roll back for this
+-- migration.
```

Comment:
nit: if there is an inverse operation, it should be added here (which isn't always the case). But the inverse of a `CREATE TABLE` would be a `DROP TABLE`.